### ISSUE TITLE
POST /signalexchange.SignalExchange/Send API frequently 

### ISSUE THREAD

**Issue by xiaolei0125** [netbirdio/netbird#1344]
**Describe the problem**
I use nginx proxy netbird signal,  and got  /signalexchange.SignalExchange/Send API call frequently when netbird android client disconnected

It seems that other netbird clients are still trying to exchange information with netbird client on android through signal server.

**Expected behavior**
If the management service knows which clients are offline, can make other clients stop to call/signalexchange.SignalExchange/Send API for the offline android client? 

**NetBird status -d output:**
Daemon version: 0.24.3
CLI version: 0.24.3
Management: Connected to https://nb.xxx.com:8443
Signal: Connected to https://nb.xxx.com:8443
FQDN: m1.nb.iot
NetBird IP: 100.64.0.3/16
Interface type: Kernel
Peers count: 3/4 Connected

**Comment by bcmmbaga** [comment]
Hello @xiaolei0125, this is the current behavior of the client. We have plans to optimize this in the next quarter and prevent constant attempts to connect, even if the peer is offline.

**Comment by xiaolei0125** [comment]
Any update?

**Comment by dkrhodes** [comment]
+1 we still got this prob with version v0.27.4

**Comment by xiaolei0125** [comment]
@bcmmbaga     I have created netbirds on many devices, but most of them are offline most of the time. I have received a large number of requests for these offline devices, and I think this is unreasonable.

**Comment by dkrhodes** [comment]
> @bcmmbaga I have created netbirds on many devices, but most of them are offline most of the time. I have received a large number of requests for these offline devices, and I think this is unreasonable.

That's true, we have 150 servers, when only a few of them got offline, the caddy and signal service used almost 100% CPU usage of a 4C8G server.

**Comment by thorleifjacobsen** [comment]
Any update on this? We have 2500 servers planned to install. This sounds scary?

**Comment by xiaolei0125** [comment]
still got this prob with version v0.28.2

**Comment by xiaolei0125** [comment]
@bcmmbaga  @mlsmaycon  @lixmal  
Do you have a plan when to solve this problem? 
I think when the number of devices is large, this problem becomes very obvious.

**Comment by mlsmaycon** [comment]
The signal reconnection shouldn't be an issue for connectivity as it happens automatically and it only does that, it shouldn't interrupt any existing peer connection.

We are looking at optimizing network map updates as now it is just brute force as it sends to all peers all the time there is an update even if there is no update for the peer.

**Comment by jacobalberty** [comment]
@nazarewk I'm seeing it. I just put a message about it in the slack
